src/components/AVPlayer/AVPlayer.js

```diff
@@ -376,7 +380,7 @@ class AVPlayer extends Component {
     const unitId = item?.unit?.id;
     if (unitId) {
       const {unit, item: { mediaType }, selectedLanguage, uiLanguage } = this.props;
-      this.props.chronicles.append('player-play', this.buildAppendData(item, this.state.src, this.props.media));
+      this.props.chronicles.append('player-play', this.buildAppendData(item, this.state.src, this.props));
```

Comment:
maybe somewhere at the beginning to write `{ chronicles } = this.props` and to use it everywhere? Just remember that javascript is interpreted language.

Reply:
Done